tools/update_tool

```diff
@@ -367,6 +368,14 @@ def main():
         default=False,
         help="Disable UUID masking"
     )
+    parser.add_argument(
+        "--compare-to",
+        default="HEAD~1",
+        help='git ref to compare to, default to previous commit')
+    parser.add_argument(
```

Comment:
Am I blind, or is this unused?

Reply:
It's used on lines 259, 294, 300 and 340 ^^.
It's just that args is directly passed to the other functions. Could be better though!

Reply:
It's me who is blind sorry! Now the `db_path` is used :sweat_smile: